### ISSUE TITLE
Support: What is the minimum role to delete attributes from an org event?

### ISSUE THREAD

**Issue by chennin7** [MISP/MISP#10090]
### Support Questions

What is the minimum role to delete attributes from an org event? We have a user who is an Org Admin who can create events and attributes in our instance for their org, but they are not able to delete attributes on the event once it's created.

### MISP version

2.4.194

### Operating System

Red Hat

### Operating System version

7.9

### PHP version

7.4.33

### Browser

Chrome

### Browser version

131.0.6778.109

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iglocska** [comment]
Manage organisation events role should be enough, assuming the event was created by a different user in the same org, normally an org admin should absolutely be able to do it. 

Was the event created on the same instance?

**Comment by chennin7** [comment]
@iglocska appreciate you taking a look. That's strange, it was created by the same user on the same instance. The event was done by importing a virus total event export.

**Comment by iglocska** [comment]
Hmm, maybe something to check:

Could you view the JSON version of the event? (/events/view/{id}.json)

Within the Event object itself, the fields I'd be most interested in are:

orgc_id (does it match the user's org_id?)
org_id (does it match the user's org_id?)
locked (is it set to 0?)

**Comment by chennin7** [comment]
Org_id and orgc_id do match, however locked is set to 'true'

**Comment by iglocska** [comment]
Ok, that explains it. Basically the ingestion method (the import) locks the event from editing, could you walk me through the process you're using to get the data into MISP?

**Comment by chennin7** [comment]
Yeah, so we're exporting data from virus total in MISP format. Below is an example:
`{"Event":{"uuid":"0cc6708e-57b3-4e98-af82-0a5c6fe5a338","info":"Untitled graph","Attribute":[{"uuid":"b127a098-a6b3-44d9-ad35-3917167bf262","type":"ip-dst","value":"167.172.153.196"}],"Object":[{"uuid":"e2df5e27-03a4-4dc7-95b9-8063de15c99e","name":"virustotal-graph","meta-category":"misc","template_uuid":"9b421055-b1bb-4c33-9ead-7fa3f39e2232","template_version":"1","Attribute":[{"uuid":"794c89e3-3df8-43a9-b9ad-6e06ec5a0b8b","type":"link","value":"https://www.virustotal.com/graph/null","object_relation":"permalink"}],"ObjectReference":[],"description":"Event generated from VirusTotal Graph"},{"uuid":"3a3bc06a-3364-43dd-8fda-afb5dd961c1b","name":"domain-ip","meta-category":"network","template_uuid":"43b3b146-77eb-4931-b4cc-b66c60f28734","template_version":"8","Attribute":[{"uuid":"9b87eb7f-1c79-469f-848c-da11a4322ca1","type":"ip-dst","value":"167.172.153.196","object_relation":"ip-dst"}],"ObjectReference":[]}]}}`

Then we're using it to create an event by going to event actions > import from > MISP standard > paste it in the box, then hit upload
